x-pack/docs/en/ml/troubleshooting.asciidoc

```diff
@@ -114,3 +117,38 @@ start the failing nodes you must downgrade the nodes then fix the problem per
 above.
 
 See also {stack-ref}/upgrading-elastic-stack.html[Upgrading the Elastic Stack].
+
+[[ml-stopjobs]]
+
+=== Problems deleting {ml} jobs
+
+Before you can delete a job, you must delete its {dfeeds} and close the job. If 
+you still encounter problems deleting the job, you might need to take extra 
+steps.
+
+*Symptoms:*
+
+* The {dfeed} was stopped and deleted successfully. When you try to delete the 
```

Comment:
@droberts195 Do we have a `_close?force` too on the datafeed?

Reply:
@sophiec20 Yes, it seems so per https://www.elastic.co/guide/en/elasticsearch/reference/master/ml-stop-datafeed.html

Reply:
Yes, it’s `_stop?force`